### ISSUE TITLE
What to do with Flash games?

### ISSUE THREAD

**Issue by hadess** [hadess/flatpak-games#9]
Anodyne's Adobe AIR wrapper should work as a 32-bit app, but what about pure Flash games like Samorost2, or the Flash version of Anodyne?

**Comment by Kekun** [comment]
Could they be shipped as GNOME Web web apps?

**Comment by hadess** [comment]
Nope. Latest version of Flash use the  Pepper API which isn't supported by WebKit, and Flash is unusable in anything but Chrome on Linux.
